### DISCUSSION TITLE
Drop Shadows and Layer filters

### DISCUSSION THREAD

**Discussion by CrispyNG** [keiwando/toonsquid#363] [Ideas]
Me and a few pals of mine are working on a short film in toonsquid that’s supposed to have a cel animation look. But to achieve that, we need layer filters, and other effects. Like blurring, drop shadows,  etc etc. Similar to how procreate, or adobe animate does it.  it would be really helpful to have this feature, since it would benefit a lot of creators, rather than just myself :). Hopefully you can consider this as a potential feature!

**Comment by ota-works** [comment]
This would be so lovely to see 😄
